tracer/src/Datadog.Trace/Ci/Sampling/CISampler.cs

```diff
@@ -0,0 +1,27 @@
+// <copyright file="CISampler.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using System.Collections.Generic;
+using Datadog.Trace.Configuration;
+using Datadog.Trace.Sampling;
+
+namespace Datadog.Trace.Ci.Sampling
+{
+    internal class CISampler : ISampler
+    {
+        public SamplingPriority GetSamplingPriority(Span span)
+        {
+            return SamplingPriority.UserKeep;
```

Comment:
Shouldn't this be `AutoKeep`? `UserKeep` is normally reserved for _users_ explicitly setting the sampling priority. Or did the CI team decide to use this in all languages as a special case?